tests/test_flask.py

```diff
@@ -675,3 +675,33 @@ def test_workout_list(client):
     assert returned_value.status_code == 200
 
     # TODO: need to test post requests
+
+def test_delete_user(client): 
+    """Testing the delete user route"""
+
+    # Not logged in 
+    returned_value = client.get('/delete', follow_redirects=True)
+    assert returned_value.status_code == 200
+
```

Comment:
`assert b'login' in returned_value.data`

Asserts to make sure that the word 'login' is present for the login page.